js-activity/solo_genres.js

```diff
@@ -0,0 +1,8 @@
+export function getSoleRepresentative(books){
+    const genres = books.map(book => book.genre);
+    const soloGenres = genres.filter(function(genre, index, bookGenres) {
+        return bookGenres.indexOf(genre) === index && bookGenres.lastIndexOf(genre) === index;
+    })
+
+    return soloGenres;
+}
```

Comment:
function should return the book object if it is the sole representative of that genre.